### ISSUE TITLE
Automatically pull in headshots

### ISSUE THREAD

**Issue by morkusporkus** [benjaminsampica/DynamoLeagueBlazor#73]
Currently the pulling in of headshots is done manually, when players are added to the site they should have a headshot automatically added, or a process to find missing headshots.

**Comment by TylerEvers** [comment]
I took a first crack at this with a personal blob storage containing player headshots I had set-up for a different project. Branch is  [here](https://github.com/TylerEvers/DynamoLeagueBlazor/tree/HeadshotBlobStorage). I have a few batch processes to aide in keeping it updated and it includes all positions if you ever implement kickers or IDP.

Could remove the need for you guys to track Headshots in your DB if you wanted to rely on it. 
Just threw together this quick, all tests passed but it included a change to GivenAuthenticatedAdmin_ThenAddsAPlayer()

Let me know what you think.
